src/calc.css

```diff
@@ -0,0 +1,52 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+#root {
+  padding: 300px;
+}
+
+.calc-form {
+  min-height: 300px;
+  min-width: 300px;
+  display: grid;
+  grid-template-columns: auto auto auto auto;
+}
+
+.calc-form div {
+  text-align: center;
+  font-size: 2vw;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  border: 1px solid;
+}
+
+.calc-form div:hover {
+  cursor: alias;
+}
+
+.first-row {
+  grid-column-start: 1;
+  grid-column-end: 5;
+  justify-content: end !important;
```

Comment:
- [ ] Maintaining a site that has !important scattered throughout is a nightmare. If you feel like you need to use !important, what that says to me is that you need to brush up on your [CSS specificity](http://coding.smashingmagazine.com/2007/07/27/css-specificity-things-you-should-know/).